sklearn/model_selection/_split.py

```diff
@@ -997,6 +997,16 @@ def split(self, X, y=None, groups=None):
             for train_index, test_index in cv.split(X, y, groups):
                 yield train_index, test_index
 
+    def get_n_splits(self):
+        """Returns the number of splitting iterations in the cross-validator
+
+        Returns
+        -------
+        n_splits : int
+            Returns the number of splitting iterations in the cross-validator.
+        """
+        return self.cvargs["n_splits"] * self.n_repeats
```

Comment:
Don't rely on n_splits bring present in charge even if it works for the two cases here